modules/serverless-admin-monitoring-eventing-broker-trigger.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-admin-monitoring.adoc
+
+[id="serverless-admin-monitoring-eventing-broker-trigger"]
+= Monitoring Knative Eventing brokers and triggers
+
+You can use the {product-title} monitoring dashboards to view metrics for brokers and triggers in your cluster.
+
+.Prerequisites
+
+* You have cluster administrator permissions, and access to the *Administrator* perspective in the {product-title} web console.
+* You have installed the {ServerlessOperatorName}, as well as the Knative Eventing component.
+* The {product-title} monitoring stack is enabled on your cluster. You can enable monitoring for {ServerlessProductName} during installation, by checking the box to *Enable operator recommended cluster monitoring on this namespace* when installing the {ServerlessOperatorName}.
```

Comment:
Forgot to include this in my initial review, but I think you can remove the comma in this last sentence (for each instance in the Prerequesites for these modules since it repeats):

> The {product-title} monitoring stack is enabled on your cluster. You can enable monitoring for {ServerlessProductName} during installation by checking the box to **Enable operator recommended cluster monitoring on this namespace** when installing the {ServerlessOperatorName}.